identity-account-storage/src/stronghold/client_path.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity_iota_core::did::IotaDID;
+
+#[derive(Debug, Clone)]
+pub struct ClientPath(pub String);
```

Comment:
It needs to be `pub` because the `stronghold` module is used from the `storage` module. Although that made me realize that we export our stronghold wrapper interface publicly, which is unnecessary and a lot of it will go away with the upcoming migration to the new version anyway:

```rs
// identity/src/lib.rs
#[cfg(feature = "stronghold")]
pub use identity_account_storage::stronghold::*;
```

I think we should make this a private module, and only expose our `Stronghold` type from the `storage` module. Might as well do that breaking change with the next release. (I think that should be a separate PR and ideally after this PR was merged to avoid conflicts).